### ISSUE TITLE
Adapt to latest Pharo 7 changes

### ISSUE THREAD

**Issue by jecisc** [SeasideSt/Seaside#987]
In Pharo 7 Sven rewrote the logic of base64Decoding to be right. 

This leads to the need of updating some Seaside code for Pharo 7.

This method:

```Smalltalk
seasideAdaptorRunningIcon 
	^ icons
		at: #'seasideAdaptorRunningIcon'
		ifAbsentPut:[ Form fromBinaryStream: (Base64MimeConverter mimeDecodeToBytes: self seasideAdaptorRunningIconContents readStream) ]
```

No longer works and the correct version is:

```Smalltalk
seasideAdaptorRunningIcon 
	^ icons
		at: #'seasideAdaptorRunningIcon'
		ifAbsentPut:[ Form fromBinaryStream: (self seasideAdaptorRunningIconContents base64Decoded readStream) ]
```

**Comment by jecisc** [comment]
Same for seasideAdaptorStoppedIcon

**Comment by jbrichau** [comment]
Fixed in cf11220a1889022168585dd38eec426ea38a9c23
